documentation/decisions/008-specify-quality-requirements.md

```diff
@@ -0,0 +1,22 @@
+# 008: Architects are able to specify quality requirements
+
+## Status: Accepted
+
+## Context 
+Within the FOUNDATION level working group we had long (aka _endless_) discussions about quality and quality requirements.
+Conclusion is:
+
+* Software architects shall be able to formulate precise quality requirements, i.e. as quality scenarios.
+* The shall not _invent_ these, but formulate in cooperation or collaboration with other stakeholders.
+ 
+## Rationale:
+
+* The ability to formulate such requirements in precise and practically usable ways is part of many CPSA-Advanced level certifications, therefore it *must* be taught and practiced within foundation level trainings.
```

Comment:
Would add:

* Often, there is a lack of good quality requirements in development projects. Due to the strong influence of quality requirements on the architecture, architects should be able to formulate them.

Reply:
@Ulibecker: This topic is different than the "LG 03-07 interface characteristics". 
Is it a change request for this issue or something else?

Reply:
Refer to issues #15, #299 , #396 , #539 , and most recently #597.
There are several reasons (legal, etc.) why we shouldn't do this.